### PR TITLE
#1 update cta text

### LINKED ISSUE #1: Spotlights, update Call to Action
https://trello.com/c/WjTYDAqT/753-spotlights-update-call-to-action-and-implement-scroll-arrows-sama
Update call to action.

### PR DESCRIPTION
**Pull request by samaSiriwardhanage**
Resolves #1 

I would like to indicate that visitors can select videos by clicking arrows to the next screen, and either wait for the next video to play physically move to adjacent Spotlights.

### Acceptance Criteria

- [ ] Update call to action text to TAP LENS ON READER 

### Relevant design files
https://www.figma.com/file/r8dJf66KcjkRONvMTqd8rF/VE---Gallery?node-id=729%3A13076

### Testing instructions
Add a list of steps required to test the feature.
1. See the text that matches to the design.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
As discussed, let's add some tests to make sure that our new change stays put.

Simon implemented a test you can reference here: https://github.com/ACMILabs/spotlights/pull/97/files#diff-848d5bb24167507f5fc0aca738023eb7f4ed34dbe519ef6aa75a8c7eff244508R47-R52

File: app/static/index.js [hunk 1]
```diff
@@ -160,7 +160,7 @@ for (let i = 0; i < playlist_content.length; i++) {
   const collect = document.createElement("div");
   item_inner.appendChild(collect);
   collect.className = "list_item_collect";
-  collect.innerHTML = "COLLECT";
+  collect.innerHTML = "TAP LENS ON READER";
```
Comment (ghost) [inline app/static/index.js]: Looks great!

However I can see that we also need to update the collect text here: https://github.com/samaSiriwardhanage/spotlights/blob/d6e0db14ec60e7468e5243275bd9683319ae6883/app/static/index.js#L319-L323

**Review by ghost — CHANGES_REQUESTED** [review]
Looking great! I can see one change we need to make however.

File: app/static/index.js (hunk 1 above)
Comment (ghost) [reply app/static/index.js]: @samaSiriwardhanage Have a look at how Simon implemented this by using a constant: https://github.com/ACMILabs/spotlights/pull/97/files#diff-419f549c39ea2e68a3d4d3d379938911b69065e2b3a698c5e46c5abe66d998b7R41

This way we only need to update the text in one place.

Let me know if you have any questions.

Comment (samaSiriwardhanage) [reply app/static/index.js]: @sam-maher-acmi I've added constant variable.

**Review by ghost — DISMISSED** [review]
Looks good!

Thanks for the tests.

**Review by ghost — CHANGES_REQUESTED** [review]
Looking close!

Looks like some linting is failing: https://github.com/samaSiriwardhanage/spotlights/runs/6978216992?check_suite_focus=true

Test the linting locally with `make lintjs`

**Review by ghost — APPROVED** [review]
Perfect, nice work.

**Pull request merged**